### PR TITLE
fixed the fonts for the English Illnesses page

### PR DESCRIPTION
**Pull request by patelneel1499**
fixed the fonts using pre made style from android studio

### CODE REVIEW

**Comment by mandemeskel** [comment]
Can you post the screenshots of the update here? Whenever you are doing updates to the UI you want to put screenshots so that code reviewers know what the changes look like. You can't always tell from looking at the code. Thanks!

There is a screenshot button on the emulator, it is in the white bar on the right side, it looks like a camera.

screen shot 2018-02-22 at 12 11 06 pm

**Comment by patelneel1499** [comment]
Attached 2 screen shots of 2 different illnesses for a sample but all look like these.

screenshot_1519331177
screenshot_1519331206

File: app/src/main/res/layout/clap.xml [hunk 1]
```diff
@@ -44,106 +44,82 @@
...
                 android:paddingTop="10dp"
-                /> <!--This is where the name of the page goes-->
+                android:text="Chlamydia"
+                android:textAlignment="center"
+                android:textAppearance="@style/TextAppearance.AppCompat.Headline"
+                android:textColor="#000000"
+                android:textSize="30dp"
+                android:textStyle="bold" /> <!--This is where the name of the page goes-->
             <TextView
                 android:layout_width="match_parent"
```
Comment (mandemeskel) [inline app/src/main/res/layout/clap.xml]: This code is repeated in each TextView, it can become a style.

File: app/src/main/res/layout/clap.xml [hunk 2]
```diff
@@ -44,106 +44,82 @@
...
```
Comment (mandemeskel) [inline app/src/main/res/layout/clap.xml]: This can be style, Page Name style or something.

File: app/src/main/res/layout/clap.xml [hunk 3]
```diff
@@ -44,106 +44,82 @@
...
-                /> <!--This is where the name of the page goes-->
+                android:text="Chlamydia"
+                android:textAlignment="center"
+                android:textAppearance="@style/TextAppearance.AppCompat.Headline"
+                android:textColor="#000000"
+                android:textSize="30dp"
+                android:textStyle="bold" /> <!--This is where the name of the page goes-->
             <TextView
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
```
Comment (mandemeskel) [inline app/src/main/res/layout/clap.xml]: This can also be a style, for headlines.

**Comment by patelneel1499** [comment]
fixed the changes you requested and put everything on styles

screen shot 2018-02-24 at 1 29 32 pm

screen shot 2018-02-24 at 1 29 19 pm

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -83,6 +83,44 @@
...
 
+    <style name = "IllnessesPage">
+        <item name= "android:textColor">#000000</item>
+        <item name= "android:textSize">12sp</item>
+    </style>
+
+    <style name="IllnessesPage.Title" parent="IllnessesPage">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">55dp</item>
+        <item name="android:paddingTop">10dp</item>
```
Comment (mandemeskel) [inline app/src/main/res/values/styles.xml]: All the styles have the same padding properties - paddingTop, paddingLeft, paddingRight -, so you can move that stuff into the parent style, IllnessesPage.
Once you do that feel free to merge the code in.
Great job, looks good!

**Pull request merged**